source/codegen/metadata/nidcpower/attributes.py

```diff
@@ -293,6 +293,7 @@
         'access': 'read-write',
         'channel_based': True,
         'name': 'POWER_LINE_FREQUENCY',
+        'enum': 'PowerLineFrequencies',
```

Comment:
Document all the changes from `attributes.py`, `functions.py` and `enum.py` to CHANGES.md file.

Reply:
documented the change